### ISSUE TITLE
[BUG] The network stats are empty.

### ISSUE THREAD

**Issue by m-rtijn** [FredrikAugust/server-status-remake#4]
The up / down arrows are visible, but no information is shown.

**Comment by FredrikAugust** [comment]
output from dev console

**Comment by m-rtijn** [comment]
Mongo:

```
connecting to: test
> use serverstatus
switched to db serverstatus
> db.load.find({})
>
```

App.js:

Warning: missing space before text for line 62 of jade file "/var/www/html/server-status-remake/views/index.jade"
Warning: missing space before text for line 65 of jade file "/var/www/html/server-status-remake/views/index.jade"
(And this repeated over and over again)

Loop.sh doesn't give any errors

**Comment by m-rtijn** [comment]
Here's also a picture where you can see that the network stats are empty: http://i.imgur.com/VgeO59j.png

**Comment by FredrikAugust** [comment]
Fixed in commit b265e24c7cc3244c381e2c97e5ab1662
